scripts/duck/people/reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { Constants, ActionTypes, State } from ".";
+
+const initialState: State = {
+    peopleList: [],
```

Comment:
The semicolon at the end might be dangerous. This will be interpreted as `initialState = { peopleList: [], undefined }`